src/mod/endpoints/mod_sofia/mod_sofia.h

```diff
@@ -323,6 +323,7 @@ typedef enum {
 	PFLAG_SDP_MEDIA_STRICT_FMT,
 	PFLAG_ALWAYS_BRIDGE_EARLY_MEDIA,
 	PFLAG_ENABLE_100REL_SYNC,
+	PFLAG_DISABLE_CHALLENGE_RESPONSES,
```

Comment:
Just minor suggestion, probably rename this as `PFLAG_DISABLE_AUTH_CHALLENGE_RESPONSE`